### DISCUSSION TITLE
"Suggested" question get reset after each update

### DISCUSSION THREAD

**Discussion by luisbrandao** [open-webui/open-webui#2112] [General]
# Bug Report

## Description

**Bug Summary:**
Everytime i bump the docker version to latest, the "suggest question" reset to default.

**Steps to Reproduce:**
Instal via docker, change the suggested questions, bump the version of the container.

**Expected Behavior:**
Suggested question are permanent.

## Environment

- **Open WebUI Version:** [e.g., 0.1.122~0.1.124]

- **Operating System:** [Rocky linux 8 + docker (ghcr.io/open-webui/open-webui:v0.1.124)]
- **Browser (if applicable):** [brave 1.64.116]]

**Comment by justinh-rahb** [comment]
`-v ./config.json:/app/backend/data/config.json`

**Comment by tjbck** [comment]
Related #1022

**Comment by luisbrandao** [comment]
```
docker run -d --name=open-webui \
  --restart=unless-stopped \
  --network main \
  --publish 127.0.0.1:8083:8083 \
  -e OLLAMA_BASE_URL=http://gw.brandao \
  -e PORT=8083 \
  --volume open-webui-data:/app/backend/data \
  ghcr.io/open-webui/open-webui:v0.1.124

```

Im already have the data in a volume.... should this be enought?

**Reply by justinh-rahb** [reply]
It'll be persisted, yes... but you'll not have easy ways to edit it. You can copy the file out and back in:
```bash
docker cp open-webui:/app/backend/data/config.json .
nano config.json
docker cp ./config.json open-webui:/app/backend/data/config.json
```

Otherwise, it's much more convenient to bind mount that file so that it can be edited anytime on the host:
```bash
docker run -d --name=open-webui \
  --restart=unless-stopped \
  --network main \
  --publish 127.0.0.1:8083:8083 \
  -e OLLAMA_BASE_URL=http://gw.brandao \
  -e PORT=8083 \
  --volume open-webui-data:/app/backend/data \
  --volume ./config.json:/app/backend/data/config.json \
  ghcr.io/open-webui/open-webui:main
```
